check_limits.py

```diff
@@ -1,18 +1,36 @@
 
-def battery_is_ok(temperature, soc, charge_rate):
-  if temperature < 0 or temperature > 45:
-    print('Temperature is out of range!')
-    return False
-  elif soc < 20 or soc > 80:
-    print('State of Charge is out of range!')
-    return False
-  elif charge_rate > 0.8:
-    print('Charge rate is out of range!')
-    return False
+def Temperature_check(temperature):
+    if temperature < 0 or temperature > 45:
+        print('Temperature is out of range!')
+        return False
+
+def State_of_charge_check(soc):
+    if soc < 20 or soc > 80:
+        print('State of Charge is out of range!')
+        return False
+
+def charge_rate_check(charge_rate):
+    if charge_rate > 0.8:
+        print('Charge rate is out of range!')
+        return False
+
+def battery(temperature,soc,charge_rate):
```

Comment:
naming of the method can be more intuitive. Battery just represents a object and not some action. The method should be self explanatory so that the user understands what it does.